benchmark/requirements.txt

```diff
@@ -1,4 +1,5 @@
 numpy
 opencv-python==4.5.4.58
```

Comment:
Should update to latest opencv-python. Maybe try `opencv-python<5.0`?

Reply:
Yes, sure.